elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -1,7 +1,113 @@
 defmodule Web.Settings.DNS do
   use Web, :live_view
 
+  alias Domain.Config
+
+  defp remove_errors(changeset, field, message) do
```

Comment:
We should move this function down after the function or group of functions that use this one. 

This is for developer friendliness, most of the time people going to this module will be looking at `mount/3` (which should always come first) and `render/1`. So they should be close to the top. This will allow people to scroll less and ease "loading" the module into their human memory :).

Reply:
I've never been sure of what the convention should be for these liveview files, so glad we're discussing it.  Should I strive for the following then:

```
def mount()

...

def render()

...

def handle_event()

...

defp any_other_helper_funcs()

...

```

Happy to follow whatever convention makes the most sense.

Reply:
Would it make sense to codify these conventions? https://hexdocs.pm/credo/Credo.Check.Readability.StrictModuleLayout.html

Reply:
That would be great if that's possible, because it would definitely help me in learning either elixir or even team best practices/conventions.